extensions/arc/src/localizedConstants.ts

```diff
@@ -43,7 +43,7 @@ export const saveText = localize('arc.save', "Save");
 export const discardText = localize('arc.discard', "Discard");
 export const resetPassword = localize('arc.resetPassword', "Reset Password");
 export const addExtensions = localize('arc.addExtensions', "Add extensions");
-export const dropExtension = localize('arc.dropExtension', "Drop extension");
+export const dropExtensions = localize('arc.dropExtensions', "Drop extensions");
```

Comment:
I think the verb "drop" is confusing.  I'm not sure this actually drops the extension.  It just adds/removes it from postgres' `shared_preload_libraries`.  I think the user still has to run `CREATE EXTENSION` or `DROP EXTENSION`.  Would be good to test with some extensions that require preloading like pg_cron, pgaudit.

Reply:
Will follow up on best descriptors of actions available on this page